src/controllers/controller.bar.js

```diff
@@ -190,8 +190,8 @@ module.exports = DatasetController.extend({
 
 	/**
 	 * Returns the stacks based on groups and bar visibility.
-	 * @param {Number} [last] - The dataset index
-	 * @returns {Array} The stack list
+	 * @param {number} [last] - The dataset index
+	 * @returns {object[]} The stack list
```

Comment:
I don't think objects are primitive types and this should be `Object[]` if this function was returning objects. Its actually not. Its `string[]` or could be `number[]`.
@simonbrunel's point of not maintaining documentation for private functions seems to be valid.

Reply:
[TypeScript recommends to use `object` instead](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html), but I agree that the comment is outdated.

Reply:
Updated to `string[]` to match the docs https://www.chartjs.org/docs/latest/charts/bar.html.